dsf-bpe/dsf-bpe-server-jetty/pom.xml

```diff
@@ -206,4 +141,57 @@
 			</plugin>
 		</plugins>
 	</build>
+
+	<profiles>
+		<profile>
+			<id>copy-highmed-processes</id>
```

Comment:
Is this profile still needed?

Reply:
I suggest we keep it for now. The activated copy goal makes it easy to start the bpe server from the IDE with HiGHmed processes.